contracts/RexCaptain.sol

```diff
@@ -201,12 +195,10 @@ contract REXCaptain is AccessControlEnumerable, SuperAppBase {
 
     /// @dev Check if a vote has passed. Quorum is required, and more than half of the votes must be 'for'.
     function _isVotePassed(RexCaptainStorage.Vote storage currentVote) internal view returns (bool) {
-        uint256 qMul = captains.length * 2;
-        uint256 quorum = (qMul / 3);
-        // If quorum is not a whole number, take its ceiling
-        if (qMul % 3 > 0) {
-            quorum += 1;
-        }
+
+        // TODO: Double check this, quorum should be 33% fo the captains
+        uint256 quorum = captains.length / 3;
+        console.log(quorum);
```

Comment:
1) What should be the quorum if there are 10 captains, 3 or 4?. 
2) If there are 2 captains doing this might give us the floor of 0.66 which in this case is 0.

Reply:
Console logs out 1 when captains.length == 2